src/data/finset.lean

```diff
@@ -659,6 +659,18 @@ begin
   { intro x, simp, intros hx hx₂, refine ⟨or.resolve_left (h hx) hx₂, hx₂⟩ }
 end
 
+@[simp] lemma filter_eq [decidable_eq β] (s : finset β) (b : β) :
```

Comment:
Making this lemma a `simp` lemma will stop `x \in s.filter (eq b)` simplifying to `x \in s \and x = b`

Reply:
Removed.